src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -411,8 +411,12 @@ CHIP_ERROR HKDF_SHA256(const uint8_t * secret, const size_t secret_length, const
     return error;
 }
 
-CHIP_ERROR pbkdf2_sha256(const uint8_t * password, size_t plen, const uint8_t * salt, size_t slen, unsigned int iteration_count,
-                         uint32_t key_length, uint8_t * output)
+PBKDF2_sha256::PBKDF2_sha256(void) {}
```

Comment:
nit: we generally want to use `()` instead of `(void)` (reasoning: for consistency, clang auto-fixes can fix to make () the standard but not the other way ... so we settled on the version that can be mass-made consistent).

Reply:
Sure. Also moved to Header as suggested by the other reviewer.